packages/cactus-plugin-ledger-connector-aries/src/main/typescript/index.web.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
@outSH: Is the API client intentionally omitted from the web exports? Whatever you export here will be accessible to the webpack bundles in the front-end packages, e.g. right now we couldn't use the api client in an angular/react app.

Reply:
Added. This can be done in bulk for other packages as well (most don't export anything as well)

Reply:
@outSH Thank you! Agreed, if the other connector plugins also don't export the API clients for web then that's definitely something we should address before v2.0.0 goes out!